homeassistant/components/peblar/strings.json

```diff
@@ -51,7 +51,7 @@
         "data_description": {
           "password": "[%key:component::peblar::config::step::user::data_description::password%]"
         },
-        "description": "Set up your Peblar EV charger to integrate with Home Assistant.\n\nTo do so, you will need the password you use to log into the Peblar EV charger' web interface.\n\nHome Assistant will automatically configure your Peblar EV charger for use with Home Assistant."
+        "description": "Set up your Peblar EV charger at {host} to integrate with Home Assistant.\n\nTo do so, you will need the password you use to log into the Peblar EV charger' web interface.\n\nHome Assistant will automatically configure your Peblar EV charger for use with Home Assistant."
```

Comment:
This is well, I just wonder if we should store the whole `ZeroconfServiceInfo` (instead of just the host) and use something like 

`hostname (IP address)` instead?

Reply:
In that case we maybe could use the more friendly hostname, (PBLR-<something>) which is more recognisable?

Reply:
Yes, that is what I meant:

`PBLR-0000645 (10.10.11.56)`

Reply:
Or even write it out

`Set up your Peblar EV charger PBLR-0000645, on IP address 10.10.11.56, to integrate with Home Assistant`

Reply:
That would be nice!

Will do that, but it will take a day or two longer due some reasons 🎄.

Reply:
I can pick up if you are ok with that?

Reply:
Yeah sure!